Bands RFC #1213 (#1254)
radiantearth/stac-spec: best-practices.md

| Spatial extent                 | Limited                                             | Full                                                               | Full                                                                                         |
| Use case                       | Quick overview, often in lists of items/collections | Display for a single Item/Collection, sometimes shown on a web map | Display for a single Item/Collection, often shown on a map, may be displayed in GIS software |

### Bands

> The new `bands` array in combination with the property inheritance introduced in STAC 1.1 provides users with more flexibility.

Reviewer: It's new now, but it won't always be.

```suggestion
As of STAC 1.1, the `bands` array can be used in combination with property inheritance to provide users with more flexibility.
```
